recipes-kernel/kernel-tests/kernel-performance-tests-files/cyclictest-container/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:22.04
```

Comment:
Can we use a "latest" tag here instead of "22.04"? (re: https://hub.docker.com/_/ubuntu)
Trying to avoid having to manually rev this version in the future.